legislators-social-media.yaml

```diff
@@ -4482,3 +4482,69 @@
   social:
     facebook: RepGianforte
     youtube_id: UCj4ocM2PHNKBbmMQF8k-wsQ
+- id:
+    bioguide: B001306
+    govtrack: 412747
+  social:
+    twitter: RepBalderson
+- id:
+    bioguide: A000376
+  social:
+    twitter: RepColinAllred
+- id:
+    bioguide: B001306
+    govtrack: 412747
+  social:
+    twitter: RepBalderson
```

Comment:
There's a duplicate entry here for Balderson, but this otherwise looks good.

Reply:
Good catch.